Tasks/PublishSymbols/IndexHelpers/DbghelpFunctions.ps1

```diff
@@ -1,26 +1,30 @@
+function Get-DbghelpPath {
+    [CmdletBinding()]
+    param()
+
+    Trace-VstsEnteringInvocation $MyInvocation -Parameter @()
+    $dbghelpPath = [System.IO.Path]::GetFullPath("$PSScriptRoot\..\dbghelp.dll")
+    Assert-VstsPath -LiteralPath $dbghelpPath -PathType Leaf -PassThru
```

Comment:
`-passthru` means the value will flow into the output pipeline. Also, below you `return`. Does the caller get an array with the file path twice?

note, there is tracing already in assert-vstspath and since you call it in one place, another possibility would be to inline this below as one line. your decision, just fyi.